README.md

```
@@ -1,38 +1,189 @@
+<p float="left">
+  <img src="homework-screenshots/9_listening_arabic-quiz-app.png" width="400" />
+  <img src="homework-screenshots/10_listening_quiz-questions.png" width="400" /> 
+</p>
+<p align="center">
+  <img src="homework-screenshots/11_listening_dictation-mode-answer.png" width="400" />
+</p>
+
+### Arabic Writing Practice
+
+<p align="center">
+  <img src="homework-screenshots/12_arabic-writing-app.png" width="400" />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)